### PR TITLE
What's new initial

### PR DESCRIPTION
**Pull request by girayimprobable**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
Initial version of what's new
#### Tests
How did you test these changes prior to submitting this pull request?
What automated tests are included in this PR?
#### Documentation
How is this documented (for example: release note, upgrade guide, feature page, in-code documentation)?
#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

File: SpatialGDK/Documentation/content/whatsnew.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+*If you’ve used the pre-alpha GDK you may be interested in these latest changes.*
+
+## Iterative improvements
+- Upgraded the engine fork to support Unreal Engine 4.20.3.
+- Upgraded to the new SpatialOS load balancer, providing fast multi-worker iteration.
+- Removed the incremental build time overhead of using the GDK by implementing dynamic type bindings, avoiding the need to generate certain interop code. You can find more info [here]({{urlRoot}}/content/dynamic-typebindings)
+- Schema generation for all classes is now the default.
+- Clients no longer automatically connect to a deployment, unless you pass it a commandline argument. For example: 
+```"%UNREAL_HOME%\Engine\Binaries\Win64\UE4Editor-Win64-Debug.exe" "%~dp0%PROJECT_PATH%\%GAME_NAME%.uproject" 127.0.0.1```
```
Comment (girayimprobable) [inline SpatialGDK/Documentation/content/whatsnew.md]: Not sure why this is not rendering correctly. Due to the %?

File: SpatialGDK/Documentation/content/whatsnew.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# What's New in Alpha?
+
+*If you’ve used the pre-alpha GDK you may be interested in these latest changes.*
```
Comment (ElleEss) [inline SpatialGDK/Documentation/content/whatsnew.md]: Ditch the stars.

File: SpatialGDK/Documentation/content/whatsnew.md (hunk 1 above)
Comment (ElleEss) [reply SpatialGDK/Documentation/content/whatsnew.md]: Do the three backticks need to be on a different line to the text to top and tail it?

File: SpatialGDK/Documentation/content/whatsnew.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# What's New in Alpha?
+
+If you’ve used the pre-alpha GDK you may be interested in these latest changes.
+
+## Iterative improvements
+- Upgraded the engine fork to support Unreal Engine 4.20.3.
+- Upgraded to the new SpatialOS load balancer, providing fast multi-worker iteration.
+- Removed the incremental build time overhead of using the GDK by implementing dynamic type bindings, avoiding the need to generate certain interop code. You can find more info [here]({{urlRoot}}/content/dynamic-typebindings)
+- Schema generation for all classes is now the default.
+- Clients no longer automatically connect to a deployment, unless you pass it a commandline argument. For example:
```
Comment (Vatyx) [inline SpatialGDK/Documentation/content/whatsnew.md]:
```suggestion
- Clients no longer automatically connect to a deployment when not in PIE, unless you pass it a commandline argument. For example: 
```

**Pull request merged**